h2/src/main/org/h2/mvstore/rtree/MVRTreeMap.java

```diff
@@ -27,7 +28,7 @@
     /**
      * The spatial key type.
      */
-    final SpatialDataType keyType;
+    private final SpatialDataType keyType;
```

Comment:
The default vilibility was correct, this field is accessed from inner classes.

Reply:
I do not see any such usages. This field shadows field from parent class.

Reply:
`findIntersectingKeys()` and `findContainedKeys()` have anonymous inner classes that read this field.

Reply:
ops, I see